### DISCUSSION TITLE
SD MODELS & HARD DISC SPACE

### DISCUSSION THREAD

**Discussion by tmprabubiz** [lllyasviel/stable-diffusion-webui-forge#1936] [General]
These 2 questions are meant for experts and contributors of this forum

1. How you people manage to test various SD/SDXL/SD3?FLUX Models, Loras, Embeddings and Extensions with the hard disc you have. I have a dedicated 2TB SSD for WebUI's, Models and others said above. Still, I am afraid of HDD space issues. (I use 200MB/S broadband and 8GB VRAM and 64GB CPU.

2. How to hire cloud GPUs for my pc? What are the best sources and how much they charge?

**Comment by wisdombox44** [comment]
watch this guy he has alotof tutorials how to run wvarious webuis on cloud
https://www.youtube.com/@SECourses

**Comment by HMRMike** [comment]
I just delete what I don't like if several attempts don't even come near what I needed. About 250GB taken so far, mainly with some SDXL models, Flux D fp16, and Q8, text ncoders in Q8 and fp16, and a bunch of controlnet models for XL, but most will be deleted as Union ProMax outperforms them, also some LORAs come and go.
